### DISCUSSION TITLE
Dịch "soft alignment"

### DISCUSSION THREAD

**Discussion by MarkusKhoa** [d2l-ai/d2l-vi#2659] [Terminology]
Theo các anh/em trong group thì nên dịch chữ `soft alignment` này thế nào ạ? Em có nên dịch là `sắp xếp gọn` không?

cc: @minhduc0711 @cuongvng @lkhphuc @davidnvq @rootonchair

**Comment by lkhphuc** [comment]
@MarkusKhoa cho mình thêm ngữ cảnh xuất hiện được không?
Từ `soft...` như `softmax` là chỉ phiên bản `có thể tính đạo hàm / khả vi` của operator gốc như là `max`, nên mình nghĩ từ này cũng vậy.

**Reply by MarkusKhoa** [reply]
Em xin trích đoạn: `In the next step, we compare a word in one sequence with the other sequence that is softly aligned with that word. 
Note that in soft alignment, all the words from one sequence, 
though with probably different attention weights, will be compared with a word in the other sequence.` Từ này xuất hiện trong những chương NLP ạ.
